.github/workflows/build-publish-next.yml

```diff
@@ -13,15 +13,11 @@ on:
   push:
     branches:
       - master
-  # schedule:
-  #   - cron: '0 0 * * *' # once a day at midnight
-  workflow_dispatch:
+  schedule:
```

Comment:
are you sure to want to enable nightlies instead of just next build ?
nightly builds have been disabled on ci.centos as it could update the image with invalid behavior (not passing happy path for example)

Reply:
You're right. I forgot about that.
I've updated. Thanks!